examples/Components/Dropdown.purs

```diff
@@ -2,24 +2,22 @@ module Components.Dropdown where
 
 import Prelude
 
-import Effect.Aff (Aff)
 import Data.Array ((!!), mapWithIndex, length)
+import Data.Const (Const)
 import Data.Maybe (Maybe(..), fromMaybe)
 import Data.Monoid (guard)
+import Data.Tuple.Nested ((/\))
+import Effect.Aff (Aff)
 import Halogen as H
 import Halogen.HTML as HH
+import Halogen.Hooks (useLifecycleEffect, useState)
+import Halogen.Hooks as Hooks
+import Halogen.Hooks.Extra.Hooks (useEvent)
 import Internal.CSS (class_, classes_, whenElem)
+import Select (SelectReturn(..), selectInput, useSelect)
 import Select as S
-import Select.Setters as SS
-
-type Slot =
-  H.Slot S.Query' Message
 
-type State =
-  ( items :: Array String
-  , selection :: Maybe String
-  , buttonLabel :: String
-  )
+type Slot query = H.Slot query Message
```

Comment:
This is `Const Void` below for the query type

```suggestion
type Slot = H.Slot (Const Void) Message
```

Reply:
Done.